e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/policy-and-exports/change-your-answers/change-your-answers-policy-and-exports-single.spec.js

```diff
@@ -0,0 +1,295 @@
+import partials from '../../../../../partials';
+import { FIELD_IDS, ROUTES } from '../../../../../../../constants';
+import { submitButton } from '../../../../../pages/shared';
+import { checkYourAnswersPolicyAndExports } from '../../../../../pages/insurance/check-your-answers';
+import application from '../../../../../../fixtures/application';
+import { singleContractPolicyPage } from '../../../../../pages/insurance/policy-and-export';
+import {
+  checkChangeLinkUrl,
+  changeAnswerField,
+  checkChangeAnswerRendered,
+} from '../../../../../../support/check-summary-list-field-change';
+import { changeAnswerSelect } from '../../../../../../support/check-summary-list-select-change';
+import formatCurrency from '../../../../../helpers/format-currency';
+import currencies from '../../../../../../fixtures/currencies';
+import { createTimestampFromNumbers, formatDate } from '../../../../../helpers/date';
+
```

Comment:
Sorry to nit pick can we rename this file name and the multiple one so it has `single-policy-type` at the end?